<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SDKDemo</class>
 <widget class="QDialog" name="SDKDemo">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SDKDemo</string>
  </property>
  <property name="windowIcon">
   <iconset theme="SDKDemo">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>411</width>
     <height>561</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Picture</string>
    </attribute>
    <widget class="QLabel" name="label_1">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>60</y>
       <width>170</width>
       <height>226</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>215</x>
       <y>60</y>
       <width>170</width>
       <height>226</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="imgtxt1">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>300</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="imgtxt2">
     <property name="geometry">
      <rect>
       <x>215</x>
       <y>300</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>330</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>사진1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>330</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>사진2</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="pictureconsole">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>390</y>
       <width>385</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pictureinit">
     <property name="geometry">
      <rect>
       <x>295</x>
       <y>20</y>
       <width>90</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Init</string>
     </property>
    </widget>
    <widget class="QComboBox" name="picmodcombo">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>20</y>
       <width>170</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>381</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btestbtn">
        <property name="text">
         <string>기본 테스트</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ptestbtn">
        <property name="text">
         <string>성능 테스트</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="extestbtn">
        <property name="text">
         <string>확장 테스트</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>CAM</string>
    </attribute>
    <widget class="QPushButton" name="caminit">
     <property name="geometry">
      <rect>
       <x>295</x>
       <y>20</y>
       <width>90</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Init</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cammodcombo">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>20</y>
       <width>170</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="camconsole">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>390</y>
       <width>385</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="camtestbtn">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>CAM 테스트</string>
     </property>
    </widget>
    <widget class="QPushButton" name="capturebtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>캡쳐</string>
     </property>
    </widget>
    <widget class="QPushButton" name="assignbtn">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>360</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>등록</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="capturetext">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>360</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="camimg">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>381</width>
       <height>301</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>CAM</string>
     </property>
    </widget>
    <widget class="QLabel" name="captureimg">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>260</y>
       <width>121</width>
       <height>91</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <zorder>camimg</zorder>
    <zorder>caminit</zorder>
    <zorder>cammodcombo</zorder>
    <zorder>camconsole</zorder>
    <zorder>camtestbtn</zorder>
    <zorder>capturebtn</zorder>
    <zorder>assignbtn</zorder>
    <zorder>capturetext</zorder>
    <zorder>captureimg</zorder>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
